packages/cli/cloudbuild.yaml

```diff
@@ -0,0 +1,18 @@
+steps:
```

Comment:
Why do we cloud build the CLI?

Reply:
It could be used to separate out the Docker image updates in the container registry.